[🐞 BUG] Custom AI provider settings are regularly lost

**Describe the bug**
Custom AI provider settings are regularly lost. When I open the settings, the entire list of providers disappears, including my custom provider, and my custom prompt is also missing. After adding a custom provider again, the previously filled custom prompts reappear.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to Preferences → AI Enhancements → Providers and Prompts & Advanced
2. Add custom provider and custom Dictation Prompt
3. After few successful dictations Fluid starts act without those settings and if you open Preferences → AI Enhancements again it will be empty (even without predefined providers)

**Expected behavior**
Stable AI Enhancements settings

**Screenshots**

<img width="1081" height="934" alt="Image" src="https://github.com/user-attachments/assets/94ee741e-637a-47b3-917e-e5b96c9ab45f" />

**Environment:**
 - macOS Version: Sequoia 15.7.3 (24G419)
 - App Version: v1.5.5
 - Architecture: Apple Silicon

**Additional context**
Add any other context about the problem here.

**Logs**
```
[18:04:10.516] [ERROR] [SettingsStore] Failed to persist provider API keys: One or more parameters passed to a function were not valid.
[18:04:18.522] [ERROR] [SettingsStore] Failed to persist provider API keys: One or more parameters passed to a function were not valid.
[18:04:18.537] [DEBUG] [AISettingsView] testAPIConnection: provider=openrouter, model=anthropic/claude-sonnet-4.5, baseURL=https://openrouter.ai/api/v1, fullURL=https://openrouter.ai/api/v1/chat/completions
```

> Oh wow. This is a pretty crazy bug for sure. Is this random? Do the builtin ones ever come back after disappearing? Can you reproduce it? I have never seen this till now. If you could help me some way to reproduce it, it would be amazing! Thanks

It's 100% stable — after any moves away from AI Enhancement window (close settings or switching tab). I suspect my custom Russian prompt, since it's the first "unusual" thing in whole setup... Could you also made logs more informative about: "[18:04:10.516] [ERROR] [SettingsStore] Failed to persist provider API keys: One or more parameters passed to a function were not valid."? Which exactly parameters?

--- a/Sources/Fluid/Persistence/KeychainService.swift
+++ b/Sources/Fluid/Persistence/KeychainService.swift
@@ -161,7 +161,6 @@ final class KeychainService {
         case errSecDuplicateItem:
             let updateAttributes: [String: Any] = [
                 kSecValueData as String: data,
-                kSecAttrAccessible as String: kSecAttrAccessibleAfterFirstUnlock,
             ]
             let updateStatus = SecItemUpdate(
                 aggregatedQuery() as CFDictionary,

